Fixed #32643 -- Fixed decoding of messages in the pre-Django 3.2 format.

apollo13 opened the pull request:
As discussed… I am not sure about release notes, I guess we do only need them for the cherry pick back to 3.2 and not main?

felixxm reviewed (approved):
@apollo13 Thanks :+1: 

I added release notes and simplified tests.

apollo13 commented:
Can you readd the previous error checking from the test? Otherwise we do not know for sure that the added message does indeed cause the exception ;)

On Wed, Apr 14, 2021, at 21:46, Mariusz Felisiak wrote:
> 
> 
> ***@***.**** approved this pull request.
> 
> @apollo13 <https://github.com/apollo13> Thanks 👍
> 
> I added release notes and simplified tests.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/django/django/pull/14266#pullrequestreview-636019314>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAT5C6VRN6PMT5ABZ7PUMTTIXWK3ANCNFSM425HLD7Q>.
>

felixxm commented:
> Can you readd the previous error checking from the test? Otherwise we do not know for sure that the added message does indeed cause the exception ;)
> […](#)
> On Wed, Apr 14, 2021, at 21:46, Mariusz Felisiak wrote: ***@***.**** approved this pull request. @apollo13 <https://github.com/apollo13> Thanks +1 I added release notes and simplified tests. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#14266 (review)](https://github.com/django/django/pull/14266#pullrequestreview-636019314)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAT5C6VRN6PMT5ABZ7PUMTTIXWK3ANCNFSM425HLD7Q>.

Agreed, added :+1:
